packages/wdio-selenium-standalone-service/README.md

```diff
@@ -71,7 +71,7 @@ export const config = {
             logPath: './temp',
             args: {
                 version: "3.141.59",
```

Comment:
This version should be bumped to v4 if you want to make this a v4 example

Reply:
and also baseURL (https://github.com/SeleniumHQ/selenium/releases/download) and https://github.com/webdriverio/selenium-standalone/blob/HEAD/docs/API.md#seleniuminstallopts (contains link to old https://selenium-release.storage.googleapis.com/index.html) need to be updated, right?